pkg/api/types.go

```diff
@@ -1007,6 +1007,23 @@ type PodSecurityContext struct {
 	// Use the host's ipc namespace.
 	// Optional: Default to false.
 	HostIPC bool `json:"hostIPC,omitempty"`
+
+	// Capabilities are the capabilities to add/drop when running containers
+	Capabilities *Capabilities `json:"capabilities,omitempty"`
+
+	// Run all containers in privileged mode
+	Privileged *bool `json:"privileged,omitempty"`
+
+	// SELinuxOptions is the SELinux context to be applied to all containers
+	SELinuxOptions *SELinuxOptions `json:"seLinuxOptions,omitempty"`
+
+	// RunAsUser is the UID to run the entrypoint of all container processes
+	RunAsUser *int64 `json:"runAsUser,omitempty"`
```

Comment:
I would prefer to make this consistent with the rest of the file even though we know it is not right. i have a laundry list of stuff in that area I want to handle once I have some bandwidth freed up and will fix that issue and writr a verifier around it.